vue/components/ui/molecules/button-dropdown/button-dropdown.vue

```diff
@@ -1,15 +1,27 @@
 <template>
     <div class="button-dropdown" v-bind:class="classes">
-        <div class="button button-primary" v-on:click="onPrimaryClick">
-            <template v-if="primaryIcon">
-                <icon class="icon icon-front" v-bind:icon="primaryIcon" v-bind:color="'#000000'" />
-                <icon class="icon icon-back" v-bind:icon="primaryIcon" v-bind:color="'#ffffff'" />
-            </template>
+        <div
+            class="button button-primary"
+            v-on:click="onPrimaryClick"
+            v-on:mouseover="primaryActive = true"
+            v-on:mouseout="primaryActive = false"
+        >
+            <icon
+                class="icon"
+                v-bind:icon="primaryIcon"
+                v-bind:color="buttonColorPrimary"
+                v-if="primaryIcon"
+            />
             <span class="label">{{ label }} </span>
         </div>
-        <div class="button button-secondary" ref="button-secondary" v-on:click="onSecondaryClick">
-            <icon class="icon icon-front" v-bind:icon="secondaryIcon" v-bind:color="'#000000'" />
-            <icon class="icon icon-back" v-bind:icon="secondaryIcon" v-bind:color="'#ffffff'" />
+        <div
+            class="button button-secondary"
+            ref="button-secondary"
+            v-on:click="onSecondaryClick"
+            v-on:mouseover="secondaryActive = true"
+            v-on:mouseout="secondaryActive = false"
+        >
+            <icon class="icon" v-bind:icon="secondaryIcon" v-bind:color="buttonColorSecondary" />

```

Comment:
Being an open source project you can't really say that. I believe that it may not have issues with our internal stuff but that doesn't mean it won't break other stuff.

You should atleast maintain those `icon-front` and `icon-back` classes as they won't affect your implementation and would still maintain some backwards compatibility.

Reply:
Changing classes inside the component will not break any retro compatibility if the component is being used correctly, thus making this point invalid.
However, I'll add it to the component so we can move on with the review 💪🏼